src/Controls/src/Core/Compatibility/Handlers/Android/FrameRenderer.cs

```diff
@@ -96,17 +97,28 @@ public FrameRenderer(Context context, IPropertyMapper mapper, CommandMapper comm
 			}
 		}
 
-		Size IViewHandler.GetDesiredSize(double widthMeasureSpec, double heightMeasureSpec)
+		Size IViewHandler.GetDesiredSize(double widthConstraint, double heightConstraint)
 		{
-			double minWidth = 20;
-			if (Primitives.Dimension.IsExplicitSet(widthMeasureSpec) && !double.IsInfinity(widthMeasureSpec))
-				minWidth = widthMeasureSpec;
+			var virtualView = (this as IViewHandler).VirtualView;
+			if (virtualView == null)
+			{
+				return Size.Zero;
+			}
```

Comment:
```suggestion
			var virtualView = (this as IViewHandler)?.VirtualView;
			if (virtualView is null)
			{
				return Size.Zero;
			}
```
As can return a null pointer, so VirtualView could be called in a null ptr and get a NRE